mscore/musescore.cpp

```diff
@@ -2812,6 +2812,23 @@ void MuseScore::showElementContext(Element* el)
       debugger->setElement(el);
       }
 
+//---------------------------------------------------------
+//   reDisplayDockWidget
+//
+//   Helper function to ensure when un-docked widgets are
+//   re-displayed, they are also re-dockable
+//---------------------------------------------------------
+
+void MuseScore::reDisplayDockWidget(QDockWidget* widget, bool visible)
+{
```

Comment:
Wrong indentation: we use 6-space indent for these braces (see other functions definitions)